addons/calendar/models/calendar_attendee.py

```diff
@@ -104,6 +105,9 @@ def _send_mail_to_attendees(self, mail_template, force_send=False):
             :param mail_template: a mail.template record
             :param force_send: if set to True, the mail(s) will be sent immediately (instead of the next queue processing)
         """
+        for event, attendees in self.grouped('event_id'):
+            if not event._do_send_mail_status_update():
```

Comment:
I would actually have the inverse method, like ``_skip_mail_status_update`` as it is clearer to me when reading (ok, this event wants to skip the emails)